rewrite/rewrite.go

```diff
@@ -107,13 +108,21 @@ func rewriteImportsInFile(fi string, rw func(string) string, rwLock *sync.Mutex)
 		return nil
 	}
 
+	offset := file.End()
+	for i, d := range file.Comments {
```

Comment:
what is this?

Reply:
Go ends up picking up an extra comment or two after the imports for some reason. Also, don't merge this yet. That commit appears to have broken something.